### PR TITLE
[zh] Translate API server config v1alpha1 reference

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by tengqm** [comment]
Preview: https://deploy-preview-31942--kubernetes-io-main-staging.netlify.app/zh/docs/reference/config-api/apiserver-config.v1alpha1/

File: content/zh/docs/reference/config-api/apiserver-config.v1alpha1.md [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+   <code>name</code> 是准入控制器的名称。此名称必须与所注册的准入插件名称匹配。
+   </p>
+</td>
+</tr>
+<tr><td><code>path</code><br/>
+<code>string</code>
+</td>
+<td>
+   <p><!--Path is the path to a configuration file that contains the plugin's configuration-->
+   <code>path</code> 为指向包含插件配置数据的配置文件的位置。
```
Comment (howieyuen) [inline content/zh/docs/reference/config-api/apiserver-config.v1alpha1.md:153]:
```suggestion
   <code>path</code> 为指向包含插件配置数据的配置文件的路径。
```

File: content/zh/docs/reference/config-api/apiserver-config.v1alpha1.md [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+<tr><td><code>name</code> <B>[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <p><!--name is the name of the egress selection.
+Currently supported values are &quot;controlplane&quot;, &quot;master&quot;, &quot;etcd&quot; and &quot;cluster&quot;
+The &quot;master&quot; egress selector is deprecated in favor of &quot;controlplane&quot;-->
+  <code>name</code> 是 Egress 选择器的名称。当前支持的取值有 &quot;controlplane&quot;，
+  &quot;master&quot;，&quot;etcd&quot; 和 &quot;cluster&quot;。
+  &quot;master&quot; Egress 选择器以被弃用，推荐使用 &quot;controlplane&quot;。
```
Comment (howieyuen) [inline content/zh/docs/reference/config-api/apiserver-config.v1alpha1.md:237]:
```suggestion
  &quot;master&quot; Egress 选择器已被弃用，推荐使用 &quot;controlplane&quot;。
```

File: content/zh/docs/reference/config-api/apiserver-config.v1alpha1.md [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+<thead><tr><th width="30%"><!--Field-->字段</th><th><!--Description-->描述</th></tr></thead>
+<tbody>
+
+<tr><td><code>caBundle</code><br/>
+<code>string</code>
+</td>
+<td>
+   <p><!--caBundle is the file location of the CA to be used to determine trust with the konnectivity server.
+Must be absent/empty if TCPTransport.URL is prefixed with http://
+If absent while TCPTransport.URL is prefixed with https://, default to system trust roots.-->
```
Comment (howieyuen) [inline content/zh/docs/reference/config-api/apiserver-config.v1alpha1.md:334]: here missing translation?

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Pull request merged**